### DISCUSSION TITLE
How to connect to a VM

### DISCUSSION THREAD

**Discussion by ghost** [cloud-hypervisor/cloud-hypervisor#5195] [Q&A]
I launch a vm via ./cloud-hypervisor but how can I interact with it via cli?
for example uploading/downloading files, executing commands, etc via a script or cli

**Comment by likebreath** [comment]
In general, you can interact with the VMM via http API calls [1] after launching a Cloud Hypervisor process. The way how you interact with the VM depends on how you configure the VM. The simplest way is using your host TTY, which is the default configuration for CH. You can also use a PTY or ssh into the guest VM with the right configurations. Our readme is a good starting point [2].

[1] https://github.com/cloud-hypervisor/cloud-hypervisor/blob/main/docs/api.md
[2] https://github.com/cloud-hypervisor/cloud-hypervisor#booting-linux
